scripts/build.js

```diff
@@ -32,7 +31,9 @@ async function buildThemes() {
       outCss = outCss.substr(1);
     }
 
-    await writeFile(DEST_DIR(sassFile.replace(/\.scss$/, '.css')), outCss, 'utf-8');
+    const outputFile = DEST_DIR(sassFile.replace(/\.scss$/, '.css').replace('src/', ''));
```

Comment:
How about replacing "src/styles" so we end up with a path like product/themes/theme.file?

Reply:
Done.